src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -421,17 +432,26 @@ void mp::LXDVMImageVault::lxd_download_image(const QString& id, const QString& s
     source_object.insert("server", stream_location);
     source_object.insert("protocol", "simplestreams");
     source_object.insert("image_type", "virtual-machine");
+    source_object.insert("fingerprint", id);
 
-    if (id.startsWith(release))
-    {
-        source_object.insert("fingerprint", id);
-    }
-    else
+    QJsonObject image_object{{"source", source_object}};
+
+    auto release = QString::fromStdString(query.release);
+
+    if (!id.startsWith(release))
     {
-        source_object.insert("alias", release);
-    }
+        QJsonObject properties_object{{"query.release", "default"},
```

Comment:
Shouldn't we use the the exact alias the user requested? So that when we update, we update for the requested alias, which could very well be "lts"?

Reply:
Good catch! This is cruft left over from testing. I need to switch it back to be `id`.

Reply:
Fixed